internal/hclogutils/args_test.go

```diff
@@ -171,3 +171,130 @@ func TestMapsToArgs(t *testing.T) {
 		})
 	}
 }
+
+func TestArgsToKeys(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]struct {
```

Comment:
Possibly add a test case for the situation where there are an uneven number of elements in `args`?